autoscheduler/scheduler/tests/scheduling_tests.py

```diff
@@ -62,11 +62,11 @@ def assert_meetings_match_expected(self, meetings, valid_sections,
             actual_section_meetings = meetings_for_sections[section]
             self.assertEqual(set(section_meetings), set(actual_section_meetings),
                              msg=f"Section {section}: found meetings {section_meetings}"
-                                 f", expected {actual_section_meetings}")
+                             f", expected {actual_section_meetings}")
 
-        # Check all sections for the course are contained in meetings
-        self.assertEqual(set(meetings), set(valid_sections),
-                         msg=f"Sections not matching: got {meetings}, "
+            # Check all sections for the course are contained in meetings
+            self.assertEqual(set(meetings), set(valid_sections),
+                             msg=f"Sections not matching: got {meetings}, "
                              f"expected {actual_section_meetings}")
```

Comment:
Change this back to how it was, the last `assertEqual` is making sure that the keys (sections) in `meetings` are equal to the expected sections, and I think keeping the extra spacing on the second line of the assert message makes it more clear that it's part of the same argument

Reply:
Sorry, my VSCode thinks this file uses 8-space tabs for some reason

Reply:
Ok so the reason I initally tabbed them over was because I got an `UnboundLocalError: local variable 'actual_section_meetings' referenced before assignment`. Any idea how to fix that?

Reply:
Yeah, I just realized that the message I print isn't accurate for that assertion, it should be:
```python
        # Check all sections for the course are contained in meetings
        self.assertEqual(set(meetings), set(valid_sections),
                         msg=f"Sections not matching: got {set(meetings)}, "
                             f"expected {set(valid_sections)}")
```